cpp/daal/include/services/internal/daal_kernel_defines.h

```diff
@@ -31,11 +32,18 @@
  * @ingroup services
  * @{
  */
+
 #define DAAL_KERNEL_SSE2
 #define DAAL_KERNEL_SSE42
 #define DAAL_KERNEL_AVX2
 #define DAAL_KERNEL_AVX512
 
+#if defined(TARGET_X86_64)
+    #include "x86_64/x86_64_kernel_defines.h"
+#elif defined(TARGET_ARM)
+    #include "aarch64/aarch64_kernel_defines.h"
```

Comment:
```suggestion
    #include "services/internal/x86_64/x86_64_kernel_defines.h"
#elif defined(TARGET_ARM)
    #include "services/internal/aarch64/aarch64_kernel_defines.h"
```
@rakshithgb-fujitsu this should resolve the error. As far as bazel changes go I don't believe they are necessary.

Reply:
Thanks!